crates/shared/src/price_estimation/factory.rs

```diff
@@ -321,6 +321,7 @@ impl<'a> PriceEstimatorFactory<'a> {
         &mut self,
         kinds: &[PriceEstimatorType],
         drivers: &[Driver],
+        spawn_update_task: bool,
```

Comment:
In general, I tend to agree with @ennmichael that these bool parameters are hard to understand at the callsite (you just see a true/false without much context).

Not blocking PR merge though.

Reply:
Sorry, didn't mean to bikeshed. As mentioned, doesn't block the PR, was just indicating personal preference.